examples/default/main.tf

```diff
@@ -19,19 +19,17 @@ data "ibm_resource_group" "existing_resource_group" {
 #############################################################################
 
 resource "ibm_resource_instance" "cos_instance" {
-  count = var.enable_vpc_flow_logs ? 1 : 0
-
-  name              = "${var.prefix}-vpc-logs-cos"
+  count             = var.enable_vpc_flow_logs ? 1 : 0
+  name              = var.prefix != null ? "${var.prefix}-vpc-logs-cos" : "vpc-logs-cos"
```

Comment:
@Khuzaima05 please remove this change. This does not have to support passing prefix as null. Infact prefix is required in the **example**